creator-node/src/snapbackSM.js

```diff
@@ -137,79 +205,99 @@ class SnapbackSM {
     return resp.data.data
   }
 
-  // Retrieve the current clock value on this node for the provided user wallet
+  /**
+   * Given wallets array, queries DB and returns a map of all users with
+   *    those wallets and their clock values
+   *
+   * @dev - TODO what happens if this DB call fails?
+   */
   async getUserPrimaryClockValues (wallets) {
+    // Query DB for all cnodeUsers with walletPublicKey in `wallets` arg array
     const cnodeUsers = await models.CNodeUser.findAll({
       where: {
         walletPublicKey: {
           [models.Sequelize.Op.in]: wallets
         }
       }
     })
-    return cnodeUsers.reduce((o, k) => {
+
+    // Convert cnodeUsers array to map (wallet => clock)
+    const cnodeUserClockValuesMap = cnodeUsers.reduce((o, k) => {
       o[k.walletPublicKey] = k.clock
       return o
     }, {})
+
+    return cnodeUserClockValuesMap
   }
 
-  // Enqueues a manual (high priority) sync.
-  // Returns the added job.
+  /**
+   * Enqueues a sync request to manualSyncQueue & returns job info
+   */
   async enqueueManualSync ({
     primaryEndpoint,
     secondaryEndpoint,
     userWallet
   }) {
     try {
       const primaryClockValue = (await this.getUserPrimaryClockValues([userWallet]))[userWallet]
-      return this.issueSecondarySync({
+
+      const jobInfo = await this.issueSecondarySync({
         userWallet,
         secondaryEndpoint,
         primaryEndpoint,
         primaryClockValue,
-        priority: SyncPriority.High,
         syncType: SyncType.Manual
       })
+
+      return jobInfo
     } catch (e) {
       logger.error(`Error enqueing manual sync for user: ${userWallet}, error: ${e.message}`)
     }
   }
 
-  // Enqueue a sync request to a particular secondary.
-  // Returns the added job
+  /**
+   * Enqueues a sync request to secondary and returns job info
+   */
   async issueSecondarySync ({
     primaryEndpoint,
     secondaryEndpoint,
     userWallet,
     primaryClockValue,
-    priority,
-    syncType = SyncType.Recurring
+    syncType
   }) {
-    let syncRequestParameters = {
+    const syncRequestParameters = {
       baseURL: secondaryEndpoint,
       url: '/sync',
       method: 'post',
       data: {
         wallet: [userWallet],
         creator_node_endpoint: primaryEndpoint,
+        // Note - `sync_type` param is only used for logging by nodeSync.js
         sync_type: syncType
       }
     }
-    // Note: we pass in syncType as job name for observability
-    return this.syncQueue.add(
-      syncType,
-      {
-        syncRequestParameters,
-        startTime: Date.now(),
-        primaryClockValue
-      },
-      { priority }
-    )
+
+    // Add job to manualSyncQueue or recurringSyncQueue based on `syncType` param
+    const jobProps = {
+      syncRequestParameters,
+      startTime: Date.now(),
+      primaryClockValue,
+      syncType
+    }
+    let jobInfo
```

Comment:
Just noticed that priority queues are actually _slower_ than queues with no priority so this is great

Reply:
From https://optimalbits.github.io/bull/ `Keep in mind that priority queues are a bit slower than a standard queue (currently insertion time O(n), n being the number of jobs currently waiting in the queue, instead of O(1) for standard queues).`

Reply:
yeee